<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInput</class>
 <widget class="QDialog" name="DialogInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px solid #595959;
background-color: #ECDC26;
border-radius: 16px;
</string>
  </property>
  <widget class="QPushButton" name="BTN_MENU_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;</string>
   </property>
   <property name="text">
    <string>미정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MENU_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;
text-align: center;
padding: 5px;</string>
   </property>
   <property name="text">
    <string>치즈볼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MENU_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;</string>
   </property>
   <property name="text">
    <string>감자튀김</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_INPUT_SENSE">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>15</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  background-color:lightyellow;
  font: 25pt &quot;함초롬돋움&quot;; 
  font-weight: bold;
  border-radius: 16px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MENU_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;</string>
   </property>
   <property name="text">
    <string>뼈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MENU_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;</string>
   </property>
   <property name="text">
    <string>순살</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>75</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>175</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>175</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>275</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>275</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;함초롬돋움&quot;;
background-color: #FFFFFF;
border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MENU_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;함초롬돋움&quot;;
font-weight: bold;
background-color: #898989;
border-radius: 16px;</string>
   </property>
   <property name="text">
    <string>미정</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>BTN_MENU_5</zorder>
  <zorder>BTN_MENU_4</zorder>
  <zorder>BTN_MENU_3</zorder>
  <zorder>LE_INPUT_SENSE</zorder>
  <zorder>BTN_MENU_2</zorder>
  <zorder>BTN_MENU_1</zorder>
  <zorder>label_7</zorder>
  <zorder>BTN_MENU_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
